modules/local/unpack_netmhc_software/main.nf

```diff
@@ -56,24 +56,21 @@ process EXTERNAL_TOOLS_IMPORT {
         -e "s_/scratch_/tmp_" \
         -e "s_setenv[[:space:]]NMHOME.*_setenv NMHOME \\`realpath -s \\\$0 | sed -r 's/[^/]+\$//'\\`_ " "${toolname}/${toolbinaryname}"
 
-    # MODIFY perl location in perl script for netmhcIIpan
-    if [ "$toolname" == "netmhciipan" ]; then
-        sed -i.bak \
-        -e 's_bin/perl.*\$_local/bin/perl_' "${toolname}/NetMHCIIpan-${toolversion}.pl"
-    fi
     #
     # VALIDATE THE CHECKSUM OF THE DOWNLOADED MODEL DATA
     #
-    checksum="\$(md5sum "$datatarball" | cut -f1 -d' ')"
-    if [ "\$checksum" != "${datachecksum}" ]; then
-        echo "A checksum mismatch occurred when checking the data file for ${toolname}." >&2
-        exit 3
-    fi
+    if [ "$toolname" == "netmhcpan" ]; then
```

Comment:
Why only for netmhcpan?

Reply:
Basically similar arguments as for syfpeithi removal. NetMHCpan is used for class I in most publications, nf-core slack didn't vote, its quite old and most like not getting an update since NetMHCpan versions use newer training data and do have the pan-allele functionality (convenience).

Reply:
I meant how `netmhcIIpan` is handled here. Not sure if you refer to the same.

Reply:
Ah! Sorry, netmhciipan-4.3 does comes with the data folder, no additional downloading needed